classify/previewer.py

```diff
@@ -66,12 +74,13 @@ def export_clip_preview(self, filename, tracker:TrackExtractor, track_prediction
         if tracker.stats:
             auto_max = tracker.stats['max_temp']
             auto_min = tracker.stats['min_temp']
-            print("Using temperatures {}-{}".format(auto_min, auto_max))
         else:
             print("Do not have temperatures to use")
             return
 
-        if track_predictions:
+        has_tracks = track_predictions and len(track_predictions) > 0
```

Comment:
This would be enough: 
```python
has_tracks = bool(track_predictions)
```

or you could avoid `has_tracks` entirely by just using `track_predictions` as a conditional. For example:
```python
show_tracks = self.preview_type == self.PREVIEW_CLASSIFIED and track_predictions
```

Reply:
thanks for the bool() trick!